src/util/metric/mod.rs

```diff
@@ -96,6 +97,9 @@ impl NonblockUdpMetricSink {
 
 impl MetricSink for NonblockUdpMetricSink {
     fn emit(&self, metric: &str) -> io::Result<usize> {
-        self.socket.send_to(metric.as_bytes(), &self.sink_addr)
+        let t = SlowTimer::from_millis(500);
+        let r = self.socket.send_to(metric.as_bytes(), &self.sink_addr);
+        slow_log!(t, "send metric too slow, takes {:?}", t.elapsed());
```

Comment:
We don't know whether send_to is really very fast and I doubt it may be slow through the log. So I use slow log here for test. I will remove this after test.